diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter5/FunctionLengthWarnTest.kt

```diff
@@ -124,6 +124,31 @@ class FunctionLengthWarnTest : LintTestBase(::FunctionLength) {
         )
     }
 
+    @Test
+    @Tag(WarningNames.TOO_LONG_FUNCTION)
+    fun `check suppress`() {
+        lintMethod(
+                """
+                    |class A() {
+                    |   val x = 10
+                    |   val y  = 11
+                    |   
+                    |   @Suppress("TOO_LONG_FUNCTION")
```

Comment:
What if suppress is on class of file level?

Reply:
Now it works